coalib/processes/Processing.py

```diff
@@ -138,7 +138,8 @@ def autoapply_actions(results,
             action = default_actions[result.origin]
         except KeyError:
             for bear_glob in default_actions:
-                if fnmatch(result.origin, bear_glob):
+                if fnmatch(result.origin, bear_glob.replace('*', '**')
+                           if bear_glob == '*' else bear_glob):
```

Comment:
Sudnt this be `if not "**" in bear_globs then replace '*' with '**' ` ?

Reply:
@newbazz wip. Please review only after I ping you😉

Reply:
actually the solution is not simple as that, I have to check for each '*' I find whether the next is also no '*' and only then I need to convert that. I am trying to find an elegant one liner for that. Also need to write a test over here.

Reply:
That is what i am telling bro, `if `**` not in x and '*' in x then replace` but then there can be something like `a*b**c` right?

Reply:
yes and I don't want to write an ugly loop

Reply:
probably you can split by '**' and then write a for loop ;P

Reply:
Then join ;)

Reply:
thats a great option!
so something like `''.join([glob.replace('*','**') for glob in bear_glob.split('**')])`

Reply:
yeah exactly!! :)

Reply:
You could also use `re.subs(r'\*[^*]', '**', text)`

Reply:
Ah maybe the regex has to be expanded to
```
\*([^*]|$)
```

Reply:
@Makman2 that is still gonna match the last `*` in any number of consecutive `*`s